driver/csiplugin/controllerserver.go

```diff
@@ -1576,21 +1565,6 @@ func (cs *ScaleControllerServer) validateSnapId(ctx context.Context, scaleVol *s
 		return status.Error(codes.Internal, fmt.Sprintf("unable to get filesystem Name for Id [%v] and clusterId [%v]. Error [%v]", sourcesnapshot.FsUUID, sourcesnapshot.ClusterId, err))
 	}
 
-	if sourcesnapshot.FsName != newvolume.VolBackendFs {
-		/*		isFsMounted, err := conn.IsFilesystemMountedOnGUINode(ctx, sourcesnapshot.FsName)
-				 if err != nil {
-					 return status.Error(codes.Internal, fmt.Sprintf("error in getting filesystem mount details for %s", sourcesnapshot.FsName))
-				 }
-				 if !isFsMounted {
-					 return status.Error(codes.Internal, fmt.Sprintf("filesystem %s is not mounted on GUI node", sourcesnapshot.FsName))
-				 }*/
-		err = checkFilesystemMountOnGUI(ctx, conn, sourcesnapshot.FsName, "", "")
-
-		if err != nil {
-			return err
```

Comment:
if we have removed the call for checkFilesystemMountOnGUI from all the places, should we remove the function definition too ?

Reply:
Yes, we have removed as it's not being used now.